src/internal/Particle.ts

```diff
@@ -23,36 +23,34 @@ import Buffer from './Buffer';
 import { CallResultsArray, LogLevel } from '@fluencelabs/avm';
 
 export class Particle {
-    id: string;
-    initPeerId: string;
-    timestamp: number;
-    ttl: number;
-    script: string;
-    signature: string;
-    data: Uint8Array;
+    signature?: string;
```

Comment:
There will always be particle signature in future. It should be calculated from it's immutable part.

Reply:
to make the signature not optional and retain the guaranties - it should be assigned in the constructor. If we can't do it now - I suggest we postpone it till later

Reply:
I mean for now the correct situation we are in is that we don't always have the signature so it's better if we can see it in types. I will add TODO for that, ok?

Reply:
Let's add a TODO. Sounds good